ArchiSteamFarm/CardsFarmer.cs

```diff
@@ -35,10 +34,17 @@
 
 namespace ArchiSteamFarm {
 	internal sealed class CardsFarmer : IDisposable {
+		internal class Game {
+			public uint AppID;
+			public float HoursPlayed;
+			public byte CardsRemaining;
```

Comment:
If you're inventing such property, make sure to update it. I think the proper place would be in ```ShouldFarm()```, and done similar to updating ```HoursPlayed```.

(This is because we report status in ```!api```, as well as making decisions based on saved status without checking all badge pages again)